### ISSUE TITLE
Invalid handling of NoneType

### ISSUE THREAD

**Issue by abudden** [Electronic-Mango/simple-justwatch-python-api#1]
I installed this module today and tried it (with python 3.10.12 as that's the system python - I know the README says python 3.11 but I don't want to update at the moment.

I found what I hope is a minor bug:

```
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from simplejustwatchapi import search
>>> result = search('Die Hard', 'GB', 'en', 10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../simplejustwatchapi/justwatch.py", line 31, in search
    return parse_search_response(response.json())
  File ".../simplejustwatchapi/query.py", line 168, in parse_search_response
    entries = [_parse_entry(node["node"]) for node in nodes]
  File ".../simplejustwatchapi/query.py", line 168, in <listcomp>
    entries = [_parse_entry(node["node"]) for node in nodes]
  File ".../simplejustwatchapi/query.py", line 184, in _parse_entry
    poster = _IMAGES_URL + content.get("posterUrl")
TypeError: can only concatenate str (not "NoneType") to str
```

Presumably the API has returned something invalid (or nothing at all) for the posterUrl.

**Comment by Electronic-Mango** [comment]
Hey, the issue should be fixed in 0.9. If JustWatch doesn't return any poster URL, then "poster" field in response from this API now will be set to None.

**Comment by abudden** [comment]
> Hey, the issue should be fixed in 0.9. If JustWatch doesn't return any poster URL, then "poster" field in response from this API now will be set to None.

Brilliant, thank you
